### PR TITLE
Export wikidata properties

### PR DESCRIPTION
**Pull request by zerebubuth**
For features in the `water`, `places` and `pois` layers, export [Wikidata](https://www.wikidata.org) IDs, where they're available from upstream data (i.e: OSM, NE or WOF).

Note: this requires https://github.com/tilezen/tilequeue/pull/375 for the WOF import to work properly, although tests should still pass without it.

Connects to #858.

### CODE REVIEW

File: data/schema.sql [hunk 1]
```diff
@@ -33,7 +33,8 @@ CREATE TABLE wof_neighbourhood (
   inception DATE NOT NULL DEFAULT '0001-01-01',
   cessation DATE NOT NULL DEFAULT '9999-12-31',
   is_visible BOOLEAN NOT NULL DEFAULT true,
-  l10n_name HSTORE
+  l10n_name HSTORE,
+  wikidata TEXT
```
Comment (nvkelso) [inline data/schema.sql]: nit: `wikidata_id`?

File: docs/layers.md [hunk 2]
```diff
@@ -743,6 +743,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 * `max_zoom`: a suggested maximum zoom beyond which the place should not be visible. Currently neighbourhoods only, from Who's On First.
 * `is_landuse_aoi`: Currently neighbourhoods only, from Who's On First
 * `kind_detail`: the original value of the OSM `place` tag and Natural Earth `featurecla`, see below.
+* `wikidata`: when present, the [Wikidata](https://www.wikidata.org) ID corresponding to this feature.
```
Comment (nvkelso) [inline docs/layers.md]: Let's export these as `wikidata_id` so it's clear what that it's a concordance value not a web link.

File: data/schema.sql (hunk 1 above)
Comment (nvkelso) [reply data/schema.sql]: Could leave as wikidata here, might make import easier. But the ETL should transform it to `wikidata_id` on export.

File: docs/layers.md [hunk 3]
```diff
@@ -827,6 +828,7 @@ To resolve inconsistency in data tagging in OpenStreetMap we normalize several o
 * `exit_to`: only for highway exits
 * `ref`: generally only for `aeroway_gate` and `station_entrance` features
 * `religion`: TODO
+* `wikidata`: when present, the [Wikidata](https://www.wikidata.org) ID corresponding to this feature.
```
Comment (nvkelso) [inline docs/layers.md]: `wikidata_id` (see above)

File: docs/layers.md [hunk 4]
```diff
@@ -2374,6 +2376,7 @@ Tilezen calculates the composite exterior edge for overlapping water polygons an
 * `area`: in square meters (spherical Mercator, no real-world), `polygon` features only
 * `id`: OpenStreetMap feature `osm_id`, when sourced from `openstreetmap.org`
 * `is_tunnel`: for `line` features only (`true` values only)
+* `wikidata`: when present, the [Wikidata](https://www.wikidata.org) ID corresponding to this feature.
```
Comment (nvkelso) [inline docs/layers.md]: `wikidata_id` (see above)

File: integration-test/1425-more-osm-features.py [hunk 5]
```diff
@@ -926,6 +926,29 @@ def test_plaque_node(self):
                 'kind': u'plaque',
             })
 
+    def test_memorial_node(self):
+        # if a memorial _isn't_ a plaque, then it should stay as a memorial!
```
Comment (nvkelso) [inline integration-test/1425-more-osm-features.py]: Worth a note in the changelog!

File: yaml/places.yaml [hunk 6]
```diff
@@ -23,6 +23,7 @@ global:
       call:
         func: util.true_or_none
         args: [ { col: is_landuse_aoi } ]
+    wikidata: {col: wikidata}
```
Comment (nvkelso) [inline yaml/places.yaml]: Export as `wikidata_id` (see above)

File: yaml/pois.yaml [hunk 7]
```diff
@@ -20,6 +20,7 @@ global:
     attraction: {col: attraction}
     zoo: {col: zoo}
     exit_to: {col: exit_to}
+    wikidata: {col: wikidata}
```
Comment (nvkelso) [inline yaml/pois.yaml]: Export as `wikidata_id` (see above)

File: yaml/water.yaml [hunk 8]
```diff
@@ -11,6 +11,7 @@ globals:
     label_placement: {col: label_placement}
     boundary: {col: boundary}
     tunnel: {col: tunnel}
+    wikidata: {col: wikidata}
```
Comment (nvkelso) [inline yaml/water.yaml]: Export as `wikidata_id` (see above)

File: integration-test/1425-more-osm-features.py (hunk 5 above)
Comment (nvkelso) [reply integration-test/1425-more-osm-features.py]: Added https://github.com/tilezen/vector-datasource/issues/1872 to remind us to put into changelog.

**Comment by zerebubuth** [comment]
> `wikidata` > `wikidata_id` and then LGTM

In the past, we've mostly followed the OSM convention when it comes to property names, which in this case would be just plain `wikidata`. However, `wikidata_id` is more explicit - fixed in e458e3a.

**Comment by nvkelso** [comment]
Not yet in a dev build.

**Pull request merged**